apps/client/src/routes/dashboards/dashboards-index/styles.css

```diff
@@ -1,11 +1,15 @@
 @import '../../../styles/variables.css';
 
 .btn-custom-primary {
- background: var(--colors-background-button) !important;
- border-color: var(--colors-background-button) !important;
+  background: var(--colors-background-button) !important;
+  border-color: var(--colors-background-button) !important;
 }
 
 .btn-custom-primary:hover {
- background: var(--colors-button-hover) !important;
- border-color: var(--colors-button-hover) !important;
-}
\ No newline at end of file
+  background: var(--colors-button-hover) !important;
+  border-color: var(--colors-button-hover) !important;
+}
+
+.dashboards-table-filter > div > div {
```

Comment:
we should not be styling the cloudscape table in this way as we cannot guarantee that this will always be the correct CSS selector 

if there is no prop defined to change the width of something in Cloudscape then we cannot modify it

Reply:
thank you!